### ISSUE TITLE
data processing problem

### ISSUE THREAD

**Issue by hanxuelian** [ComputationalAgingLab/ComputAge#64]
Has the data been processed by BMIQ? Why is there a decimal point in the age value

**Comment by shappiron** [comment]
No, data was retrieved from GEO as they are. We found this more logical due to different ways of preprocessing data for clocks construction. Thus, having all data preprocessing methods different all clocks are put in the same equal conditions for benchmarking.

The age value has a decimal point because it was converted to float data type. Cases where age has non-zero digits after a decimal point mean that the original data were in months or had these digits initially.
